qa/util/src/main/java/io/camunda/qa/util/cluster/TestStandaloneCamunda.java

```diff
@@ -68,7 +72,16 @@ public final class TestStandaloneCamunda extends TestSpringApplication<TestStand
           .withEnv("xpack.security.enabled", "false")
           .withEnv("xpack.watcher.enabled", "false")
           .withEnv("xpack.ml.enabled", "false")
-          .withEnv("action.destructive_requires_name", "false");
+          .withEnv("action.destructive_requires_name", "false")
+          .withExposedPorts(9200, 9300) // Expose the ports used by Elasticsearch
```

Comment:
You can ignore this for this review, the tests is running locally forcing on the port 9200

I am on it with @nicpuppa in order to solve the port problem on camunda client builder together.